php/src/Google/Protobuf/Internal/Message.php

```diff
@@ -61,19 +61,27 @@ class Message
     /**
      * @ignore
      */
-    public function __construct($desc = NULL)
+    public function __construct($data = NULL)
```

Comment:
I would prefer to have two parameters here to avoid confusion.

Reply:
I believe we came to an agreement to keep the data array as the first parameter.